SIM-66

leroyvandijk opened the pull request:
- Added different camera for plane and tank
- Added HUD text for different camera
- Updated gif
- Updated camera controller for better zooming
- Fixed a bug in destroyed grid tiles that are created when there is already a new building
- Fixed propellor moving for airplane
- Fixed bug with spawn point for plane
- Fixed tank rotating back
- Changed all sprites for a better background

leroyvandijk committed:
- Added first person camera for plane and tank
- Added UI icons for tank and plane for bottom bar
- Fixed propellor moving for airplane
- Fixed bug with spawn point for plane
- Fixed tank rotating back

leroyvandijk committed:
- First person HUD text
- Fixed a bug in destroying grid tiles

leroyvandijk committed:
- Updated gif
- Changed all sprites for a nicer background
- Tank or plane sprite color now changes when selected

leroyvandijk committed:
- Updated Camera Controller for better zooming
- Updated version number of inno setup file

leroyvandijk committed:
- Updated README

HarmenWehkamp commented on src/Assets/Editor/AssetsPropertiesApplier.cs:
```diff
@@ -143,6 +144,35 @@ private enum ComponentType
+									cameraObject.tag = "PlaneCamera";
+									cameraObject.transform.position = new Vector3(-0.88f, 25.1f, -4.7f);
+									cameraObject.transform.rotation = Quaternion.Euler(70, 0, 0);
+									camera.fieldOfView = 47.1f;
```
Magic number 47.1?

HarmenWehkamp replied:
A lot of random numbers here actually. You be the judge if this is future proof enough.

HarmenWehkamp replied:
Create a variable that holds the vector.

var positionForThisSpecificPurpose = new Vector3(-0.88f, 25.1f, -4.7f);
cameraObject.transform.position = positionForThisSpecificPurpose;

HarmenWehkamp commented on src/Assets/Scripts/Components/Navigators/TankNavigator.cs:
```diff
@@ -152,6 +228,10 @@ public void SetStandby(bool standby)
 			IsStandby = standby;
+			if (_enableTopText)
+			{
+				_topText.text = "Stand-by";
```
Lots of places where you change the topText. Maybe make a method that changes this text based on an enum.

leroyvandijk committed:
- Added camera manager for managing active camera
- Added TopText component to seperate code
- Added events when target changed of plane or tank
- Fixed opening new tab for WebGL version when opening neighbourhood URL

leroyvandijk committed:
- Added more comments

leroyvandijk committed:
- Removed on click for plane

leroyvandijk committed:
- Fixed opening a webpage in Unity Editor
- Fixed clicking somewhere when camera is changed
- Updated README

leroyvandijk committed:
- Remove unused main camera from OnObjectClickManager

leroyvandijk committed:
- Reset highlighting and infopanel when camera is changed
- Don't follow target when camera is not active

leroyvandijk committed:
- Splitted functions in AssetsPropertiesApplier
